architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,17 +175,51 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
+//from https://blog.openshift.com/image-streams-faq/
+An Image Stream is a pointer to an image using various identifying qualities. This means that even if the source image changes, 
+the Image Stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
```

Comment:
Unless you, either:

1. have scheduled import set
2. manually re-import the image. 

I think it's worth mentioning that.